docs/README.md

```diff
@@ -55,51 +36,75 @@ There are several optional parameters that could be passed into `CreateVolumeReq
 | "encrypted"       |                  |          | Whether the volume should be encrypted or not. Valid values are "true" or "false" | 
 | "kmsKeyId"        |                  |          | The full ARN of the key to use when encrypting the volume. When not specified, the default KMS key is used |
 
-## Prerequisites
-### Kubernetes
-1. Kubernetes 1.12+ is required. Although this driver should work with any other container orchestration system that implements the CSI specification, so far it has only been tested in Kubernetes.
-
-2. Enable the flag `--allow-privileged=true` in the manifest entries of kubelet and kube-apiserver.
-
-3. Add `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true` in the manifest entries for `kube-apiserver` and `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,CSIBlockVolume=true` in the `kubelets`. This is required to enable topology support of EBS volumes in Kubernetes, using `volumeMode: Block` and restoring volumes from snapshots.
+# EBS CSI Driver on Kubernetes
+Following sections are Kubernetes specific. If you are Kubernetes user, use followings for driver features, installation steps and examples.
 
-4. Install the `CSINodeInfo` CRD on the cluster using the instructions provided here: [Enabling CSINodeInfo](https://kubernetes-csi.github.io/docs/csi-node-info-object.html#enabling-csinodeinfo).
-
-5. Please refer to [kubernetes CSI docs](https://kubernetes-csi.github.io/docs/) for general CSI driver setup instructions on kubernetes.
+## Kubernetes Version Compability Matrix
+| AWS EBS CSI Driver \ Kubernetes Version| v1.12 | v1.13 |
+|----------------------------------------|-------|-------|
+| master branch                          | no    | yes   |
+| v0.2.0                                 | no    | yes   |
+| v0.1.0                                 | yes   | yes   |
```

Comment:
One may think that we still support (backport fixes) v0.1.0. Do we?

If not, should we signal somehow that these version are no longer maintained?

Reply:
I am not planning to backport fixes. But I still have to maintain it since EKS only supports 1.11 for now. And driver 0.2 requires 1.13. We could deprecate v0.1.0 after EKS launches 1.13.